### ISSUE TITLE
turn ticket support back on

### ISSUE THREAD

**Issue by trel** [irods-contrib/metalnx-web#56]
setting is in the config file

add/include tests

**Comment by michael-conway** [comment]
yes, the big thing to ensure is that all links are bookmarkable and support deep linking/back button, this was one of the big issues addressed in 2.0.0 but there are lots of corners to check out
